CHANGELOG.md

```diff
@@ -18,6 +18,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Bug where boundary layers would be plotted too small in 2D simulations.
+- Significant speedup for field projection computations.
+- Fix numerical precision issue in `FieldProjectionCartesianMonitor`.
```

Comment:
Could you remind me about the precision issue?

Reply:
It's linked to the backend change too. 

https://github.com/flexcompute/tidy3d/issues/986#issue-1786438059